src/cards/ha-media_player-card.html

```diff
@@ -248,8 +248,20 @@
 
   playerObjChanged: function (playerObj) {
     var picture = playerObj.stateObj.attributes.entity_picture;
+    var dummy;
     if (picture) {
-      this.$.cover.style.backgroundImage = 'url(' + picture + ')';
+      dummy = document.createElement('IMG');
```

Comment:
What about having a hidden [iron-image](https://elements.polymer-project.org/elements/iron-image) in the template? The image src could just be a computed binding to the entity_picture. We can then listen to the [loaded and error](https://elements.polymer-project.org/elements/iron-image#properties) properties and update the background image when these events fire.